server/libs/database/src/entities/asset.entity.ts

```diff
@@ -51,6 +51,12 @@ export class AssetEntity {
   @Column({ type: 'varchar', nullable: true })
   duration!: string | null;
 
+  @Column({ type: 'boolean', default: true })
+  isVisible!: boolean;
+
+  @Column({ type: 'uuid', nullable: true })
+  livePhotoVideoId!: string | null;
```

Comment:
What do you think about adding a one to one relationship to the live video and loading it as part of the asset list / get by id? Or, do you not use any data from it except the asset id to load the video?

Reply:
That is correct, so far it has been the main usage, to use the id to load the video. The secondary usage is for downloading which can also be done purely by assetId

Reply:
Sounds good.